### PR TITLE
Create makefiles for the dashboard

### LINKED ISSUE #190: Create containers with version id

### PR DESCRIPTION
**Pull request by kuronosec**
### What does this PR do?

- Resolves #190 #191 #264 #159 #290 #319 #318 #157

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: .github/workflows/push-dev-environment.yaml [hunk 1]
```diff
@@ -1,23 +1,134 @@
...
+          REACT_APP_EOS_API_NETWORK_LOGO: "https://bloks.io/img/eosio.png"
+
+      - name: Build kubernetes files
+        id: build_kubernetes_files
+        run: |
+          make \
+            build-kubernetes
+        env:
+          # general
+          NAMESPACE: jungle
```
Comment (xavier506) [inline .github/workflows/push-dev-environment.yaml:62]: we already have a namespace called `jungle` for the EOSIO nodes 

lets make a new namespace for the monitor , maybe something like `jungle-dashboard` or `jungle-monitor` ?

File: .github/workflows/push-lacchain-environment.yaml [hunk 2]
```diff
@@ -1,23 +1,128 @@
...
+
+      - name: Build and deploy kubernetes files
+        id: build_kubernetes_files
+        run: |
+          make \
+            build-kubernetes \
+            deploy-kubernetes
+        env:
+          # general
+          NAMESPACE: lacchain
```
Comment (xavier506) [inline .github/workflows/push-lacchain-environment.yaml:63]: `lacchain-dashboard`

File: .github/workflows/push-master-environment.yaml [hunk 3]
```diff
@@ -1,23 +1,125 @@
...
+
+      - name: Build and deploy kubernetes files
+        id: build_kubernetes_files
+        run: |
+          make \
+            build-kubernetes \
+            deploy-kubernetes
+        env:
+          # general
+          NAMESPACE: mainnet
```
Comment (xavier506) [inline .github/workflows/push-master-environment.yaml:62]: `mainnet-dashboard`

File: makefile [hunk 4]
```diff
@@ -84,4 +89,30 @@ start-webapp:
...
+build-kubernetes: ./kubernetes
+	@rm -Rf $(K8S_BUILD_DIR) && mkdir -p $(K8S_BUILD_DIR)
+	@for file in $(K8S_FILES); do \
+		mkdir -p `dirname "$(K8S_BUILD_DIR)/$$file"`; \
+		$(SHELL_EXPORT) envsubst <./kubernetes/$$file >$(K8S_BUILD_DIR)/$$file; \
+	done
+
+deploy-kubernetes: ##@devops Publish the build k8s files
+deploy-kubernetes: $(K8S_BUILD_DIR)
+	@for file in $(shell find $(K8S_BUILD_DIR) -name '*.yaml' | sed 's:$(K8S_BUILD_DIR)/::g'); do \
```
Comment (xavier506) [inline makefile:105]: echo "Applying kubernetes files..."

File: makefile [hunk 5]
```diff
@@ -84,4 +89,30 @@ start-webapp:
...
 
 start-logs:
-	@docker-compose logs -f hapi webapp
\ No newline at end of file
+	@docker-compose logs -f hapi webapp
+
+
+build-kubernetes: ##@devops Generate proper k8s files based on the templates
+build-kubernetes: ./kubernetes
+	@rm -Rf $(K8S_BUILD_DIR) && mkdir -p $(K8S_BUILD_DIR)
```
Comment (xavier506) [inline makefile:97]: echo "Build kubernetes files..."

**Comment by kuronosec** [comment]
Hi @xavier506 and @adriexnet, I think this is in a good shape to make another review and maybe merge on dev. Please let me know if you have questions.

**Pull request merged**